prime-router/settings/organizations.yml

```diff
@@ -1277,4 +1277,134 @@
         host: sftp
         port: 22
         filePath: ./upload
-        credentialName: DEFAULT-SFTP
\ No newline at end of file
+        credentialName: DEFAULT-SFTP
+- name: "vt-doh"
+  description: "Vermont Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "VT"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "vt-doh"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/VT/VT-receiver-transform.yml"
```

Comment:
Please use "azure:/" instead of "classpath:/".  We only use classpath:/ for localhost test.